### PR TITLE
Refactor fill-scenarios for v2

### PR DESCRIPTION
**Pull request by fabioberger**
## Description
This PR:
- refactors @0xproject/fill-scenarios for V2 of the protocol
- Removes the legacy compact-artifacts from @0xproject/order-utils and adds a missing CHANGELOG entry to the package.

## Motivation and Context
Part of our tooling update for V2.

## How Has This Been Tested?

It cannot be tested until we update a package that uses fill-scenarios in their tests. This will occur while refactoring contract-wrappers.

## Types of changes

*   [ ] Bug fix (non-breaking change which fixes an issue)
*   [ ] New feature (non-breaking change which adds functionality)
*   [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [x] Added new entries to the relevant CHANGELOG.jsons.
*   [ ] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [ ] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
-        "postinstall": "rm -rf `pwd`/packages/order-utils/node_modules/@0xproject; mkdir `pwd`/packages/order-utils/node_modules/@0xproject; ln -s `pwd`/packages/json-schemas `pwd`/packages/order-utils/node_modules/@0xproject/json-schemas; ln -s `pwd`/packages/types `pwd`/packages/order-utils/node_modules/@0xproject/types; rm -f `pwd`/packages/contracts/node_modules/@0xproject/types; ln -s `pwd`/packages/types `pwd`/packages/contracts/node_modules/@0xproject/types"
+        "postinstall": "rm -rf `pwd`/packages/order-utils/node_modules/@0xproject; mkdir `pwd`/packages/order-utils/node_modules/@0xproject; ln -s `pwd`/packages/json-schemas `pwd`/packages/order-utils/node_modules/@0xproject/json-schemas; ln -s `pwd`/packages/types `pwd`/packages/order-utils/node_modules/@0xproject/types; rm -f `pwd`/packages/contracts/node_modules/@0xproject/types; ln -s `pwd`/packages/types `pwd`/packages/contracts/node_modules/@0xproject/types; mkdir -p `pwd`/packages/fill-scenarios/node_modules/@0xproject; ln -s `pwd`/packages/types `pwd`/packages/fill-scenarios/node_modules/@0xproject/types; ln -s `pwd`/packages/order-utils `pwd`/packages/fill-scenarios/node_modules/@0xproject/order-utils"
```
Comment (LogvinovLeon) [inline package.json]: Sheesh!

File: packages/fill-scenarios/src/index.ts [hunk 2]
```diff
@@ -46,156 +47,170 @@ export class FillScenarios {
...
         const exchangeInstance = new ExchangeContract(
             artifacts.Exchange.abi,
-            signedOrder.exchangeContractAddress,
+            signedOrder.exchangeAddress,
             this._web3Wrapper.getProvider(),
             this._web3Wrapper.getContractDefaults(),
         );
 
-        const [orderAddresses, orderValues] = formatters.getOrderAddressesAndValues(signedOrder);
+        const orderWithoutExchangeAddress = {
```
Comment (LogvinovLeon) [inline packages/fill-scenarios/src/index.ts]: Is this just to remove one property?
If so - use `_.omit`

**Comment by LogvinovLeon** [comment]
Just one small comment and LGTM

File: package.json (hunk 1 above)
Comment (fabioberger) [reply package.json]: I know dude...

**Pull request merged**